github-deployments: Filter environments (#2632)
backstage/community-plugins: workspaces/github-deployments/plugins/github-deployments/README.md

);
```

> You can filter for environemt names via a environment parameter.

Reviewer: ```suggestion
You can filter for environment names via an environment parameter.
```
